system-tests/projects/no-specs-custom-pattern/cypress.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  component: {
+    supportFile: false,
+    specPattern: 'src/**/*.cy.{js,jsx}',
+    devServer: require('@cypress/react/plugins/load-webpack'),
+    devServerConfig: {
+      webpackFilename: 'webpack.config.js',
+    },
+  },
+  e2e: {
+    supportFile: false,
```

Comment:
Oops, looks like you updated this to be false but missed deleting the empty support files